commands/apps_test.go

```diff
@@ -189,109 +189,6 @@ func TestRunAppsDelete(t *testing.T) {
 	})
 }
 
-func TestRunAppsCreateDeployment(t *testing.T) {
-	withTestClient(t, func(config *CmdConfig, tm *tcMocks) {
-		appID := uuid.New().String()
-		deployment := &godo.Deployment{
-			ID:   uuid.New().String(),
-			Spec: &testAppSpec,
-			Services: []*godo.DeploymentService{{
-				Name:             "service",
-				SourceCommitHash: "commit",
-			}},
-			Cause: "Manual",
-			Progress: &godo.DeploymentProgress{
-				PendingSteps: 1,
-				RunningSteps: 0,
-				SuccessSteps: 0,
-				ErrorSteps:   0,
-				TotalSteps:   1,
-
-				Steps: []*godo.DeploymentProgressStep{{
-					Name:      "name",
-					Status:    "pending",
-					StartedAt: time.Now(),
-				}},
-			},
-			CreatedAt: time.Now(),
-			UpdatedAt: time.Now(),
-		}
-
-		tm.apps.EXPECT().CreateDeployment(appID, true).Times(1).Return(deployment, nil)
-
-		config.Args = append(config.Args, appID)
-		config.Doit.Set(config.NS, doctl.ArgAppForceRebuild, true)
-
-		err := RunAppsCreateDeployment(config)
-		require.NoError(t, err)
-	})
-}
-
-func TestRunAppsCreateDeploymentWithWait(t *testing.T) {
-	withTestClient(t, func(config *CmdConfig, tm *tcMocks) {
-		appID := uuid.New().String()
-		deployment := &godo.Deployment{
-			ID:   uuid.New().String(),
-			Spec: &testAppSpec,
-			Services: []*godo.DeploymentService{{
-				Name:             "service",
-				SourceCommitHash: "commit",
-			}},
-			Cause: "Manual",
-			Phase: godo.DeploymentPhase_PendingDeploy,
-			Progress: &godo.DeploymentProgress{
-				PendingSteps: 1,
-				RunningSteps: 0,
-				SuccessSteps: 0,
-				ErrorSteps:   0,
-				TotalSteps:   1,
-
-				Steps: []*godo.DeploymentProgressStep{{
-					Name:      "name",
-					Status:    "pending",
-					StartedAt: time.Now(),
-				}},
-			},
-			CreatedAt: time.Now(),
-			UpdatedAt: time.Now(),
-		}
-		activeDeployment := &godo.Deployment{
-			ID:   uuid.New().String(),
-			Spec: &testAppSpec,
-			Services: []*godo.DeploymentService{{
-				Name:             "service",
-				SourceCommitHash: "commit",
-			}},
-			Cause: "Manual",
-			Phase: godo.DeploymentPhase_Active,
-			Progress: &godo.DeploymentProgress{
-				PendingSteps: 1,
-				RunningSteps: 0,
-				SuccessSteps: 0,
-				ErrorSteps:   0,
-				TotalSteps:   1,
-
-				Steps: []*godo.DeploymentProgressStep{{
-					Name:      "name",
-					Status:    "pending",
-					StartedAt: time.Now(),
-				}},
-			},
-			CreatedAt: time.Now(),
-			UpdatedAt: time.Now(),
-		}
-
-		tm.apps.EXPECT().CreateDeployment(appID, false).Times(1).Return(deployment, nil)
-		tm.apps.EXPECT().GetDeployment(appID, deployment.ID).Times(1).Return(activeDeployment, nil)
```

Comment:
This would now be expected to be called twice:

```
tm.apps.EXPECT().GetDeployment(appID, deployment.ID).Times(2).Return(activeDeployment, nil)
```